lib/cartopy/tests/mpl/test_gridliner.py

```diff
@@ -114,6 +114,14 @@ def test_gridliner():
                         top=1 - delta, bottom=0 + delta)
 
 
+@ImageTesting(['gridliner_orthographic'],
+              tolerance=0.2)
```

Comment:
Why the non-default tolerance?

Reply:
Don't know!
So I removed the tolerance keyword.